docs/README.md

```diff
@@ -0,0 +1,325 @@
+We expect these configuration files to go away relatively quickly, but for now
+EVE is still stuck with at least *DevicePortConfig/global.json* and
+*GlobalConfig/global.json* and they are documented in [legacy configuration](CONFIG.md).
+See [the following FAQ entry](FAQ.md) for how to manage both of these
+legacy files.
+
+### Runtime Configuration Properties
+
+In addition to highly structured portions of the configuration object, EVE
+allows for a flat namespace of random configuration properties that control
+various settings in a running instance of EVE. From a Controller perspective
+you can apply these types of settings to either and individual instance of
+EVE or to all instances of EVE grouped together in a project. For examples,
+if you are using ZEDEDA's Controller with zcli you will issue the following
+zcli command to set a configuration property named key to a given value:
+
+```bash
+   zcli [device|project] update <name> [--config=<key:value>...]
+```
+
+Or to give a more specific example, here's how you can allow ssh access to
+the device for debugging issues:
+
+```bash
+zcli device update myqemu --config="debug.enable.ssh:`cat .ssh/id_rsa.pub`"
+```
+
+Most of the runtime configuration properties apply to an entire running EVE
+instance. A set of property names, however, is reserved to only affected a
+particular microservice within a running instance of EVE. These reserved names
+always start with debug.*microservice*.
+
+You can find a complete list of generic and microservices specific configuration
+properties in [configuration properties table](CONFIG-PROPERTIES.md).
+
+It goes without saying that runtime configuration properties can also be
+delivered out-of-band just like any other part of the EVE's configuration
+object. While in transition from legacy configuration files, you will have
+to use [GlobalConfig/global.json](FAQ.md) to make them available to EVE.
+
+## Installing EVE on Edge Nodes
+
+EVE expects to be installed into a persistent storage medium (disk, flash, etc.)
+of Edge Nodes. While it is possible to have a live image of EVE running off of
+a  removable media (USB stick, CDROM, etc.) it isn't typically very useful. EVE
+currently does *not* support network [PXE booting](https://en.wikipedia.org/wiki/Preboot_Execution_Environment)
+and given that Edge Nodes are typically deployed with very rudimentary networks
+attached to them it is very unlikely that EVE will support that type of
+booting. That said, it may be possible to use EVE's installer in a network boot
+scenario to install EVE on Edge Nodes when they first do a network boot in a lab
+or provisioning environment.
+
+During the installation phase, EVE assumes the target storage medium to be either
+empty or have been formatted with [GPT](https://en.wikipedia.org/wiki/GUID_Partition_Table)
+that EVE can recognize and extend. Either way, after succcesfull installation
+you will find EVE claiming the following entries int the GPT:
```

Comment:
"EVE will have created the following entries in the GPT"

Reply:
fixed